### PR TITLE
Add Speed Booster and Powergrip offworld models

### PR DESCRIPTION
**Pull request by dyceron**
- Power Grip (AM2R)
- Speed Booster (AM2R, Dread)

image

### CODE REVIEW

File: src/open_samus_returns_rando/files/schema.json [hunk 1]
```diff
@@ -665,7 +665,6 @@
                 "ITEM_RESERVE_TANK_LIFE",
                 "ITEM_RESERVE_TANK_MISSILE",
                 "ITEM_RESERVE_TANK_SPECIAL_ENERGY",
-                "ITEM_OFFWORLD",
```
Comment (ThanatosGit) [inline src/open_samus_returns_rando/files/schema.json:668]: I'm in a rush: Why is this removed? We still have a generic offworld model which is the RDV logo icon thing, don't we?

File: tests/test_files/item_models_test.json [hunk 2]
```diff
@@ -697,8 +697,8 @@
             "resources": [
                 [
                     {
-                        "item_id": "ITEM_OFFWORLD",
-                        "quantity": 1
```
Comment (ThanatosGit) [inline tests/test_files/item_models_test.json:701]: Same question here (even though this is just a test file for us)

File: src/open_samus_returns_rando/files/schema.json (hunk 1 above)
Comment (dyceron) [reply src/open_samus_returns_rando/files/schema.json:668]: Because I remembered that offworld models use `ITEM_NONE`, so this item_id is technically never used. Obviously we could use it in the tests, but None with 0 quantity works just fine.

Comment (dyceron) [reply src/open_samus_returns_rando/files/schema.json:668]: Generic offworld is unaffected

Comment (ThanatosGit) [reply src/open_samus_returns_rando/files/schema.json:668]: Oh, OK. So, it is `ITEM_NONE` just with a different model. Totally makes sense

**Review by ThanatosGit — APPROVED** [review]

File: src/open_samus_returns_rando/files/schema.json (hunk 1 above)
Comment (dyceron) [reply src/open_samus_returns_rando/files/schema.json:668]: Yep
